config/src/cli.rs

```diff
@@ -283,30 +296,41 @@ pub struct RelayerConfig {
     pub external_fee_addr: Option<Address>,
     /// When set, the relayer will automatically redeem new fees into its wallet
     pub auto_redeem_fees: bool,
-    /// The price reporter from which to stream prices.
-    /// If unset, the relayer will connect to exchanges directly.
-    pub price_reporter_url: Option<Url>,
     /// The relayer fee whitelist
     ///
     /// Specifies a mapping of wallet IDs to fees for the relayer
     pub relayer_fee_whitelist: Vec<RelayerFeeWhitelistEntry>,
 
-    // -----------------------
-    // | Environment Configs |
-    // -----------------------
-    /// The chain that the relayer settles to
-    pub chain_id: Chain,
-    /// The address of the contract in the target network
-    pub contract_address: String,
+    // ---------------------
+    // | External Services |
+    // ---------------------
     /// The address of the compliance service to use for wallet screening. If
     /// not configured, wallet screening is disabled
     ///
     /// The API of the compliance service must match that defined here:
     ///     https://github.com/renegade-fi/relayer-extensions/tree/master/compliance/compliance-api
-    pub compliance_service_url: Option<String>,
+    pub compliance_service_url: Option<Url>,
     /// The URL to export relayer events to.
     /// If omitted, the event manager is disabled.
     pub event_export_url: Option<Url>,
```

Comment:
nit: if we have the `event_export_url` under the `External Services` heading here, we should do so in the `Cli` struct above as well